floaters/src/main/java/com/mdg/droiders/floaters/FloatingViewContainer.java

```diff
@@ -15,23 +17,33 @@
     private View collapsedView;
     private View expandedView;
     private RelativeLayout.LayoutParams cachedRelativeParams;
+    private ImageView playButton, nextButton,
```

Comment:
@suyashmahar This suggestion will increase the length of code. Is it alright to follow it?